auto_round/autoround.py

```diff
@@ -162,6 +162,7 @@ def __init__(
             device_map: Union[str, dict] = None,
             super_bits: int = None,
             super_group_size: int = None,
+            float_zp: bool = False,
```

Comment:
When the user only exports to GGUF using the q*1 format, switch to float zero points directly (add a new datatype), no need to add additional arguments to the API and wrapperlinear. If gptq/awq are also exported, using int zp